machines/check_for_venv_path.sh

```diff
@@ -0,0 +1,8 @@
+# This script checks if the MLLAM_VENV_PATH is set and if not, it exits with an error message.
+
+if [ -z "$MLLAM_VENV_PATH" ]; then
+  echo "Please set the MLLAM_VENV_PATH environment variable to path of the virtual environment where you have neural-lam etc installed"
+  exit 1
```

Comment:
Nice, this will cause the following stages to not be run if the venv is not set.